src/model/time_entry.cc

```diff
@@ -434,16 +434,22 @@ void TimeEntry::LoadFromJSON(Json::Value data) {
 
     if (data.isMember("wid")) {
         SetWID(data["wid"].asUInt64());
+    } else if (data.isMember("workspace_id")) {
+        SetWID(data["workspace_id"].asUInt64());
     } else {
         SetWID(0);
     }
     if (data.isMember("pid")) {
         SetPID(data["pid"].asUInt64());
+    } else if (data.isMember("project_id")) {
+        SetPID(data["project_id"].asUInt64());
     } else {
         SetPID(0);
     }
     if (data.isMember("tid")) {
         SetTID(data["tid"].asUInt64());
+    } else if (data.isMember("project_id")) {
```

Comment:
there should be `task_id` here.

lucky to have found this, didn't notice while reading the code, accidentally discovered during testing

Reply:
I suppose we can also write tests for this in `app_test.cc`

Reply:
oh, I just realized this particular line is already merged to master in the previous PR

Reply:
pushed this fix to `master`